bin/Ref_Maker

```diff
@@ -92,6 +94,7 @@ sub main { ##no critic (Subroutines::ProhibitExcessComplexity)
     my %subs = ();
     $subs{'blat'}   = \&blat;
     $subs{'smalt'}  = \&smalt;
+    $subs{'longranger'}  = \&longranger;
     $subs{'picard'} = \&picard;
     $subs{'npgqc'}  = \&gc_frac;
```

Comment:
To help with reading, it would be nice to re-align this listing, ie make all '=' to align.